### ISSUE TITLE
Error: Unexpected usage at e.loadForeignModule 

### ISSUE THREAD

**Issue by xavArtley** [vanillawc/wc-monaco-editor#5]
Hello I was just trying the demo on https://vanillawc.github.io/wc-monaco-editor/demo/index.html
And looking at the jsconsole I get this error:
```
Error: Unexpected usage
    at e.loadForeignModule (index.min.js:1)
    at index.min.js:1
```
each time I enter a new character

**Comment by evanplaice** [comment]
Thanks for reporting this

The error occurs because Monaco fails to side-load the service worker modules. Looks like the error occurs when the web component is loaded from another domain.

I have updated the demo to load the source relative to the web component (ie the same domain), fixing the issue and reflecting actual usage.

In a prod environment where the web component and all of it's assets are located on the same server, this error should not be present.

If the only choice is to load the web component from another domain, the editor should still work despite the errors. It will do all the language-server-protocol (ie syntax analysis, etc) on the main thread instead of offloading the work to a service worker.

**Comment by evanplaice** [comment]
Fixed in [v1.4.5](https://github.com/vanillawc/wc-monaco-editor/releases/tag/v1.4.5)
